### ISSUE TITLE
Do not try to parse git data without gz support 

### ISSUE THREAD

**Issue by nijel** [phpmyadmin/phpmyadmin#12030]
If PHP doesn't have gz support we still try to parse .git object data and get a failure:

```
NOTICE: PHP message: PHP Fatal error:  Uncaught Error: Call to undefined function gzuncompress() in /home/nijel/phpmyadmin/phpmyadmin/libraries/Config.class.php:494
Stack trace:
#0 /home/nijel/phpmyadmin/phpmyadmin/libraries/display_git_revision.lib.php(26): PMA_Config->checkGitRevision()
#1 /home/nijel/phpmyadmin/phpmyadmin/index.php(87): PMA_printGitRevision()
#2 {main}
  thrown in /home/nijel/phpmyadmin/phpmyadmin/libraries/Config.class.php on line 494
```

**Comment by muhakh** [comment]
As I understand we need to add an else statement after the elseif in libraries/Config.php line 456 that raises an exception.
Am I right?

**Comment by nijel** [comment]
Yes, just add another early return from the function...

**Comment by zaverichintan** [comment]
@muhakh are you working on it?

**Comment by muhakh** [comment]
Yes

**Comment by nijel** [comment]
Sorry, it seems this error has been already fixed in 91177009afa5e49281ef900f8ffc702aab60e0c4 (issue #11476).
